src/repositories/implementations/Prisma/LoginAdminPrismaRepository.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaClient } from '@prisma/client';
+import { AdminDTO, ILoginAdminRepository } from '../../interfaces';
+import { PrismaPostgresClient } from '../../../helpers/database/PrismaPostgresClient';
+
+export class LoginAdminPrismaRepository
+    extends PrismaPostgresClient
+    implements ILoginAdminRepository
+{
+    private prisma: PrismaClient;
+
+    constructor() {
+        super();
+        this.prisma = super.getPrismaClient();
+    }
+
+    async getAdminInfoByEmail(email: string): Promise<AdminDTO> {
```

Comment:
We should add some way to check if the admin exists, maybe by returning null from the repository?

like
```ts
if(!adminUser) return null;
```
would also need changes in the UseCases, I can do that if you want